examples/encdec/encdec.py

```diff
@@ -135,8 +138,11 @@ def train(encdec, trainer, prefix, best_valid_ppl):
     train_trg_corpus = load_corpus(TRG_TRAIN_FILE, trg_vocab)
     valid_src_corpus = load_corpus(SRC_VALID_FILE, src_vocab)
     valid_trg_corpus = load_corpus(TRG_VALID_FILE, trg_vocab)
+    test_src_corpus = list(open(SRC_TEST_FILE))
+    test_ref_corpus = list(open(REF_TEST_FILE))
```

Comment:
Ditto, but need a change: replacing UNK IDs to a *never-generated* ID (e.g., `-1`) to prevent BLEU contamination.